variables.tf

```
@@ -172,6 +189,12 @@ variable "sqs_encryption_key_arn" {
   description = "The ARN of the KMS encryption key to be used for SQS (Required when `sqs_encryption_enabled` is `true`)"
 }
 
+variable "use_s3_bucket_notification" {
+  type        = bool
+  default     = false
+  description = "Set this to true to use S3 bucket notifications, rather than CloudTrail. Default behavior uses CloudTrail"
```

Can we reword this a little.  Something like this.

```suggestion
  description = "Set this to `true` to use S3 bucket notifications instead of CloudTrail. When set to `false` CloudTrail will be used. Defaults to `false`"
```